src/modules/aws_iam@0.0.1/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Role as AWSRole } from '@aws-sdk/client-iam'
+
+import { Role, } from './entity'
+import { AWS,  } from '../../services/gateways/aws'
+import { Context, Crud, Mapper, Module, } from '../interfaces'
+import * as metadata from './module.json'
+
+export const AwsIamModule: Module = new Module({
+  ...metadata,
+  utils: {
+    roleMapper: async (role: AWSRole, ctx: Context) => {
+      const out = new Role();
+      out.arn = role.Arn;
+      out.name = role.RoleName ?? '';
+      out.policyDocument = role.AssumeRolePolicyDocument ?? '';
+      const client = await ctx.getAwsClient() as AWS;
+      out.policyArns = await client.getRolePolicyArns(out.name);
```

Comment:
why this? it supposed that the input param `role: AWSRole` must have the arn value

Reply:
yes, I let me use that

Reply:
oh I think I was confused. ok if the role policy earns does not come from AWS when retrieving the role and you need a second call I guess this is ok